src/AppBundle/Controller/NoteController.php

```diff
@@ -66,6 +66,25 @@ public function getNotesAction(Request $request, ParamFetcherInterface $paramFet
     }
 
     /**
+     * Presents the form to use to create a new note.
```

Comment:
@Ener-Getick this is a subtle BC break it seems .. without moving the method up, the order of the routes was created so that the "new" route came after the "get" which resulted in a 404 ..